docker-compose.benefit.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.8"
```

Comment:
This could be moved inside `docker-compose.helsinki-benefit.yml`

Reply:
we will have 1 file, not 2. we will merge into one when backend team makes it work.